app/code/Magento/InventoryCatalog/view/adminhtml/ui_component/product_form.xml

```diff
@@ -111,7 +112,7 @@
                 </field>
                 <field name="source_status" formElement="input" sortOrder="30">
                     <settings>
-                        <elementTmpl>ui/dynamic-rows/cells/text</elementTmpl>
+                        <elementTmpl>Magento_InventoryCatalog/components/cell-status</elementTmpl>
```

Comment:
Why do we need to change the template? (especially when we don't have this template in `InventoryCatalog` module)